tests/core/pyspec/eth2spec/test/phase_1/block_processing/test_process_crosslink.py

```diff
@@ -0,0 +1,101 @@
+from eth2spec.test.context import (
+    with_all_phases_except,
+    spec_state_test,
+    always_bls,
+)
+from eth2spec.test.helpers.attestations import (
+    get_valid_on_time_attestation,
+)
+from eth2spec.test.helpers.crosslinks import (
+    run_crosslinks_processing,
+)
+from eth2spec.test.helpers.shard_block import build_shard_block
+from eth2spec.test.helpers.state import next_epoch, next_slot, next_slots
+
+
+@with_all_phases_except(['phase0'])
+@spec_state_test
+@always_bls
+def test_basic_crosslinks(spec, state):
+    next_epoch(spec, state)
+    next_epoch(spec, state)
+    state = spec.upgrade_to_phase1(state)
+    next_slot(spec, state)
+
+    committee_index = spec.CommitteeIndex(0)
+    shard = spec.compute_shard_from_committee_index(state, committee_index, state.slot)
+    body = b'1' * spec.MAX_SHARD_BLOCK_SIZE
```

Comment:
I think it'd be easier to read if we use our common `b'\x'` notation (e.g. `b'\x42'`). This is clear that it is a full byte whereas the `b'1'` makes me a bit unsure the bitlength of "1" (without looking it up or testing it myself)